sdk/src/types/block/address/bech32.rs

```diff
@@ -18,9 +18,11 @@ use packable::{
 
 use crate::types::block::{address::Address, ConvertTo, Error};
 
+const HRP_MAX: u8 = 83;
```

Comment:
Is this on-purpse `u8` instead of `usize`? If it were `usize` you'ld have less casts?

Reply:
On purpose, protocol wants an u8